### ISSUE TITLE
translation glitch in 3.1b2 bundled esl-stranslate

### ISSUE THREAD

**Issue by alicedb2** [EddyRivasLab/hmmer#133]
Spotted something when trying to reproduce 6 frame translation in Biopython and using esl-stranslate as comparison. Posting this here rather than on the easel github because it concerns the latest hmmer binaries linked on hmmer.org.

Using the esl-stranslate miniapp bundled with hmmer 3.1b2 binaries (linux and macos) I get (spurious?) TCG -> L and TCC -> F

```
echo -e '>foo\nTCGTCC\n' | esl-stranslate -r 1 -
>foo_s0_1to6
LF
>foo_s1_2to4
R
>foo_s2_3to5
V
>foo_s0_rc_5to0
GR
>foo_s1_rc_4to2
D
>foo_s2_rc_3to1
T
```

This seems inconsistent wrt most (all?) codon tables where TCG -> S and TCC -> S.

The issue appears fixed in the lastest version of easl miniapps:

```
echo -e '>foo\nTCGCGC\n' | ./esl-translate -l 1 -
>orf1 source=foo coords=2..4 length=1 frame=2  
R
>orf2 source=foo coords=3..5 length=1 frame=3  
V
>orf3 source=foo coords=1..6 length=2 frame=1  
SS
>orf4 source=foo coords=5..3 length=1 frame=5  
D
>orf5 source=foo coords=4..2 length=1 frame=6  
T
>orf6 source=foo coords=6..1 length=2 frame=4  
GR
```

**Comment by cryptogenomicon** [comment]
This is a serious bug in `esl-stranslate`. The genetic code table in the version of Easel that is bundled in HMMER 3.1b2 has typos and is incorrectly translating, shockingly enough. The bug was first reported in May 2015 and I fixed it then in our development code (`esl-stranslate` was removed; the miniapp is now `esl-translate`). However, since our move to Harvard in summer '15, I still haven't been able to find time to do a new release of HMMER3 with the bugfix. In the meantime, the workaround is to use `esl-translate` in the Easel that comes with [Infernal 1.1.2](http://eddylab.org/infernal), or use Easel from our github master branch. I'll try to release HMMER3.2 soon, including this and other bug fixes.
